### PR TITLE
Add html5 charset attribute to meta

### PR DESCRIPTION
**Pull request by frkinta**
Add [charset](https://www.w3.org/TR/2010/WD-html-markup-20101019/meta.charset.html) attribute to meta and disable content for this attribute

### CODE REVIEW

File: framework/Web/UI/WebControls/TMetaTag.php [hunk 1]
```diff
@@ -17,8 +17,8 @@
  * TMetaTag represents a meta tag appearing in a page head section.
  * You can set its {@link setID ID}, {@link setHttpEquiv HttpEquiv},
  * {@link setName Name}, {@link setContent Content}, {@link setScheme Scheme}
- * properties, which correspond to id, http-equiv, name, content, and scheme
- * attributes for a meta tag, respectively.
+ * , {@link setCharset Charset} properties, which correspond to 
+ * id, http-equiv, name, content, and scheme  * attributes for a meta tag, respectively.
```
Comment (ctrlaltca) [inline framework/Web/UI/WebControls/TMetaTag.php]: `charset` needs to be added to this list

File: framework/Web/UI/WebControls/TMetaTag.php [hunk 2]
```diff
@@ -141,8 +161,11 @@ public function render($writer)
 			$writer->addAttribute('http-equiv',$this->_httpEquiv);
 		if($this->_scheme!=='')
 			$writer->addAttribute('scheme',$this->_scheme);
+		if($this->_charset!=='')
+			$writer->addAttribute('charset',$this->_charset);
+		      if ($this->_charset === '')
```
Comment (ctrlaltca) [inline framework/Web/UI/WebControls/TMetaTag.php]: The last row of this changeset looks like a bad copypaste.

**Comment by ctrlaltca** [comment]
Also, feel free to add a short description of the change and your credits in https://github.com/pradosoft/prado/blob/master/HISTORY

**Comment by frkinta** [comment]
Thank you for the review. Changes mades. 
Don't blame me since this is my first ever PR :)

**Comment by ctrlaltca** [comment]
Thank you for the PR!

**Pull request merged**